pkg/secrets/fetch_secret_test.go

```diff
@@ -96,6 +96,7 @@ func TestExecCommandError(t *testing.T) {
 	setCorrectRight(secretBackendCommand)
 	secretBackendTimeout = 2
 	_, err = execCommand(inputPayload)
+	secretBackendTimeout = 5
```

Comment:
Instead of setting the value to `5` you should ideally get the value before updating it.
```go
secretBackendTimeoutOld = secretBackendTimeout
defer func() {
  secretBackendTimeout = secretBackendTimeoutOld
}
secretBackendTimeout = 2
```